### PR TITLE
Update UG with minimum required Node version

### LINKED ISSUE #848: Improve user guide to include required minimum node version
**Tell us about your environment**

* **MarkBind Version:** 2.4.0
* **Node Version:** 7.2.0

**What did you do? Please include the actual source code causing the issue.**

```html
markbind init
```

**What did you expect to happen?**

Get initialization Success.

**What actually happened? Please include the actual, raw output.**
image

### PR DESCRIPTION
**Pull request by hcwong**
**What is the purpose of this pull request? (put "X" next to an item, remove the rest)**

• [x] Documentation update

Fixes #848.

**What is the rationale for this request?**
Markbind had issues with old Node versions as stated in 7.2.0

**What changes did you make? (Give an overview)**
Just added an additional line in the user guide.

**Proposed commit message: (wrap lines at 72 characters)**
Update UG with minimum Node version

### CODE REVIEW

File: docs/userGuide/gettingStarted.md [hunk 1]
```diff
@@ -13,6 +13,7 @@
   %%{{ icon_ticked }}%% a basic knowledge of [Markdown](https://www.markdownguide.org/basic-syntax/) and [HTML](https://www.w3schools.com/html/) syntax<br>
   %%{{ icon_ticked }}%% a basic knowledge of running CLI commands<br>
   %%{{ icon_ticked }}%% a recent version of [npm](https://www.npmjs.com/get-npm) installed
+  %%{{ icon_ticked }}%% [Node v8.11.3](https://nodejs.org) or higher installed
```
Comment (acjh) [inline docs/userGuide/gettingStarted.md]: - `[Node v8.11.3](https://nodejs.org)` → `[Node.js](https://nodejs.org) v8.11.3`

- Node.js v8 is reaching End-of-life on 2019-12-31: https://nodejs.org/en/about/releases/
  That's less than 3 weeks from now.
  Let's recommend v10.13.0 (LTS) or higher; v10 will only reach End-of-life on 2021-04-30.

Comment (hcwong) [reply docs/userGuide/gettingStarted.md]: 👍. Fixed thanks!

Comment (yamgent) [reply docs/userGuide/gettingStarted.md]: @hcwong the version number should be outside of the link, which is what the first bullet point by @acjh meant.

Comment (hcwong) [reply docs/userGuide/gettingStarted.md]: Oops, sorry got it

Comment (acjh) [reply docs/userGuide/gettingStarted.md]: Also, `Node` → `Node.js`.

File: docs/userGuide/gettingStarted.md [hunk 2]
```diff
@@ -13,6 +13,7 @@
   %%{{ icon_ticked }}%% a basic knowledge of [Markdown](https://www.markdownguide.org/basic-syntax/) and [HTML](https://www.w3schools.com/html/) syntax<br>
   %%{{ icon_ticked }}%% a basic knowledge of running CLI commands<br>
   %%{{ icon_ticked }}%% a recent version of [npm](https://www.npmjs.com/get-npm) installed
+  %%{{ icon_ticked }}%% [Node.js](https://nodejs.org) v10.13.0 or higher installed
```
Comment (acjh) [inline docs/userGuide/gettingStarted.md]: Oh, we need a `<br>` on the line before.

**Pull request merged**